### DISCUSSION TITLE
Conversion into config file for EVCC (http) and MQTT issue

### DISCUSSION THREAD

**Discussion by Hergesheim** [niltrip/powerocean#9] [General]
Hi guys, is it possible to convert this data into a suitable EVCC preset for the powerocean? Or are you able to give a hint, how to make all integrated sensors accesible via MQTT in home assistant? Regards!

**Comment by niltrip** [comment]
Hi, you can have a look at https://github.com/tolwi/hassio-ecoflow-cloud/issues/251#issuecomment-2311199012

There is an implementation in ioBroker to get it via mqtt in HA.
